application/src/ext/java/org/opentripplanner/ext/siri/updater/azure/AbstractAzureSiriUpdater.java

```diff
@@ -270,18 +397,17 @@ protected void defaultErrorConsumer(ServiceBusErrorContext errorContext) {
       errorContext.getEntityPath()
     );
 
-    if (!(errorContext.getException() instanceof ServiceBusException)) {
+    if (!(errorContext.getException() instanceof ServiceBusException e)) {
       LOG.error("Non-ServiceBusException occurred!", errorContext.getException());
       return;
     }
 
-    var e = (ServiceBusException) errorContext.getException();
     var reason = e.getReason();
 
     if (
       reason == ServiceBusFailureReason.MESSAGING_ENTITY_DISABLED ||
-      reason == ServiceBusFailureReason.MESSAGING_ENTITY_NOT_FOUND ||
-      reason == ServiceBusFailureReason.UNAUTHORIZED
+        reason == ServiceBusFailureReason.MESSAGING_ENTITY_NOT_FOUND ||
+        reason == ServiceBusFailureReason.UNAUTHORIZED
```

Comment:
I think this should break the prettier autoformatter check. I don't know why the build pipeline doesn't fail

Reply:
fixed